scripts/pre_commit_linter.py

```diff
@@ -990,7 +991,16 @@ def _check_comments(all_files):
 
 
 def _check_docstrings(all_files):
-    """This function ensures that docstrings end in a period."""
+    """This function ensures that docstrings end in a period and the arg
+    order in the function definition matches the order int the doc string.
+
+    Args:
+        all_files: list. Names of files to consider during docstring check.
```

Comment:
Could we specify the type of object in the list? E.g. `list(str).`

Ditto below for `Returns:` section.